### PR TITLE
Teams - Module 4: FY2019Q4 Quarterly refresh

### PR DESCRIPTION
**Pull request by andrewconnell**

### CODE REVIEW

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md [hunk 1]
```diff
@@ -80,67 +84,55 @@ This exercise introduces the Yeoman generator and its capabilities for scaffoldi
...
 1. Type `yo teams` and press **Enter**.
 
-    ![Screenshot of Yeoman Teams generator.](../../Images/Exercise1-01.png)
-
 1. When prompted, accept the default **teams-app-1** as your solution name and press **Enter**.
 
 1. Select **Use the current folder** for the file location and select **Enter**. The next set of prompts asks for specific information about your Microsoft Teams app:
     - Accept the default **teams app1** as the name of your Microsoft Teams app project and press **Enter**.
     - Enter your name and press **Enter**.
+    - Accept the default selection of **v1.4** for the manifest version you would like to use and press **Enter**.
```
Comment (clearab) [inline Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md]: Should just be "Accept the default selection for the manifest version" we're on 1.5 already, and will be on 1.6 before the next refresh

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md [hunk 2]
```diff
@@ -10,16 +10,27 @@ Head on over to [official documentation](https://msdn.microsoft.com/en-us/micros
...
 ## Building the manifest
 
+To create the Microsoft Teams Apps manifest, run the `manifest` Gulp task. This will generate and validate the package and finally create the package (a zip file) in the `package` folder. The manifest will be validated against the schema and dynamically populated with values from the `.env` file.
+
 ``` bash
 gulp manifest
 ```
+## Working with the devPreview manifest version
+
+You can change the manifest version from the standard `1.3` to `devPreview` by editing the `manifest.json` file and the `manifestVersion` property. The schema validation currently supports `1.3` and `devPreview`, for other options you need to manually edit the `gulpfile.js` file and add the version and schema in the `SCHEMAS` constant. When validating the manifest you will get an error if any other versions are used and you will also get a warning if you have not manually updated the schema reference in the `manifest.json` file (which is recommended for in-editor schema validation).
```
Comment (clearab) [inline Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md]: I would say "You can change the manifest version to `devPreview`...", and remove the references to v1.3 here.

File: Teams/04 Fundamentals of Microsoft Teams/Demos/03-tab-graph/solution/README.md [hunk 3]
```diff
@@ -1,88 +1,107 @@
...
-
+# teams app1
+
+Generate a Microsoft Teams application.
+
+TODO: Add your documentation here
+
+## Getting started with Microsoft Teams Apps development
+
+Head on over to [official documentation](https://msdn.microsoft.com/en-us/microsoft-teams/tabs) to learn how to build Microsoft Teams Tabs.
```
Comment (clearab) [inline Teams/04 Fundamentals of Microsoft Teams/Demos/03-tab-graph/solution/README.md]: Link should be: https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/tabs/tabs-overview

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md (hunk 1 above)
Comment (pschaeflein) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md]: The "default" comment is in regards to the [Yeoman generator](https://github.com/officedev/generator-teams). The generator provides the following choices for the manifest version:

image

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md (hunk 2 above)
Comment (pschaeflein) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md]: This file is created by the Yeoman generator. 

We do not use preview releases for the Training Content, so perhaps this readme file gets deleted?

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md (hunk 1 above)
Comment (clearab) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md]: Right, but the Yeoman generator will support version 1.5 in a very short amount of time, and probably 1.6 before the next refresh of the content. No reason to say more than "accept the default selection for the manifest version"

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md (hunk 2 above)
Comment (clearab) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md]: Sorry, didn't notice the file path for this one. Go ahead and ignore.

File: Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md (hunk 1 above)
Comment (pschaeflein) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md]: Ah, right. I thought you wanted language around 1.5 in the instructions. Updated to 

"Accept the default selection for the manifest version..."

File: Teams/04 Fundamentals of Microsoft Teams/Demos/03-tab-graph/solution/README.md (hunk 3 above)
Comment (pschaeflein) [reply Teams/04 Fundamentals of Microsoft Teams/Demos/03-tab-graph/solution/README.md]: Made this edit. (Also changed same link in demos/01-teams-app-yeoman/solution/README.md)

**Comment by pschaeflein** [comment]
Updated PR submitted

**Pull request merged**